meetings/2018-02-20 Index2018.md

```diff
@@ -0,0 +1,188 @@
+  similar questions and if they found a solution already. If I still cannot find anything, 
+  I will file an issue on the github and ask people who is the owner of that. 
+* Mihai: Even if it’s a problem that someone else found and solved it, 
+  would still be good to report it to Node.js project to understand which version is affected. 
+* Question from room: about continuous local storage. Have you tried async … as an alternative to that? 
+* Joel: We ran into similar problems. 
+  That is the kind of feedback that we would like. 
+  It is still experimental but if it’s something where it’s not actually solving the problem 
+  would be good to know now because once it’s out of experimental it will be harder to change the APIs. 
+* Dan: When do things fall off the visibility when you go through your locks? 
+* Joel: As far as locks, other than the continuous local storage and transactions stuff, 
+  our locks are standard. 
+  We use Bunyan as transport and our locks go to Splunk. We use Bunyan to write our logs as JSON text onto file or disc. 
+* Question from the room: For any performance issues, how do find out which part is affected? 
+* Joel: We monitor a couple of basic metrics, like latency, incoming requests, the time it take 
+  to respond to incoming requests and we also log service calls. This is where the transaction 
+  comes in as you make a service call. 
+* How do you act if it’s above the threshold? 
+* Joel: to determine that it’s a problem with the SLA, when our site is continuously running, 
+  we always have a list of stacks that we know when we are operating in parameters. 
+  We continuously look at that day over day, hour over hour graphs. We plot our graphs continuously
+  in our NOC (Network Operating Center), we look like NASA. Any time the graph shows what is the nominal 
+  value and you get called, at 3am. Everyone gets called, everyone gets together and tries to find what’s going on. 
+  * We try to figure out what is causing the graph to go up, which service is not performing well.
+  Typically, more than half of the time the back-end is having an issue. 
+  Sometimes our cloud infra may have problems.  And if we find anyone of those, we fix them. 
+  It is rare that it is caused by a bug in Node.js. Typically, it happens after a new deployment. 
+* Dan: We’ve asked you a bunch of questions. Now turning the tables, do you have questions of the Node.js project?
+* Joel: Walmart Labs is using a lot of open source and is on Node.js for a few years.
+  Is there a process that Walmart Lab can look into contributing back? 
+* Tierney: Having people from the Node.js foundation or from larger companies,
+  like IBM, who have done that may be helpful for us to get together as a group 
+  under the Comm-Comm and discuss.
+* Michael: from my perspective, that is a good idea because IBM is a big company
+  and has many people come our and can look across the projects and help on getting 
```

Comment:
and I've often helped people look across the projects and helped them figure out where to get started.